front-end/src/components/player-page/game-summary/team-info.tsx

```diff
@@ -47,7 +46,7 @@ function TeamInfo(teamInfoProps: TeamInfoProps) {
                 width="15px"/>
             </Box>
             <Box sx={{ overflow: 'hidden' }}>
-              <Link to={`/player/${encodeURIComponent(player.playerName)}`}>
+              <Link to={`/player/${encodeURIComponent(player.playerName)}-${encodeURIComponent(player.tagline)}`}>
```

Comment:
Is there a function to do this? Or can we add one incase we ever need to change the symbol